astronomer/providers/google/cloud/operators/dataproc.py

```diff
@@ -96,24 +97,30 @@ def execute(self, context: Context) -> None:  # type: ignore[override]
                 raise
             self.log.info("Cluster already exists.")
 
-        end_time: float = time.time() + self.timeout
-
-        self.defer(
-            trigger=DataprocCreateClusterTrigger(
-                project_id=self.project_id,
-                region=self.region,
-                cluster_name=self.cluster_name,
-                end_time=end_time,
-                metadata=self.metadata,
-                delete_on_error=self.delete_on_error,
-                cluster_config=self.cluster_config,
-                labels=self.labels,
-                gcp_conn_id=self.gcp_conn_id,
-                impersonation_chain=self.impersonation_chain,
-                polling_interval=self.polling_interval,
-            ),
-            method_name="execute_complete",
+        cluster = hook.get_cluster(
+            project_id=self.project_id, region=self.region, cluster_name=self.cluster_name
         )
+        if cluster.status.state == cluster.status.State.RUNNING:
+            self.log.info("Cluster created.")
+            return Cluster.to_dict(cluster)
```

Comment:
We do not do the the same in deferrable mode. This would get saved as XCOM result. If we wan't to have it here, we should also add in the the deferrable mode's output and verify that the XCOM results are consistent.

Reply:
Sure. Let me check what would be the best.

Reply:
perfect. Sorry, did not check this.